onnxruntime/core/providers/openvino/backend_utils.cc

```diff
@@ -69,6 +69,43 @@ CreateCNNNetwork(const Provider_ModelProto& model_proto, const GlobalContext& gl
     ORT_THROW(log_tag + "[OpenVINO-EP] Unknown exception while importing model to nGraph Func");
   }
 
+  ValidateCNNNetwork(global_context, subgraph_context, const_outputs_map, ng_function);
```

Comment:
This method would not be needed if we are creating the CNNNetwork in one place.

Reply:
yeah right. Done.